edgedb/datatypes/hash.c

```diff
@@ -24,79 +24,6 @@
 #include "datatypes.h"
 
 
-#if PY_VERSION_HEX >= 0x03080000
```

Comment:
You can also now remove the "Portions Copyright 2001-2018" notice from the top comment in this file (it was there for the hashing function)